### PR TITLE
Rm team update

### PR DESCRIPTION
**Pull request by Hisma**
## Description

Updated the "team" page to include an "about me" for myself.  
https://flowforge.com/team/

note - I used a personal headshot that is missing the flowforge background.  An art request is in to take care of this here - https://github.com/flowforge/website/issues/845
My current headshot can be replaced once the art request is completed.  

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/flowforge/flowforge/blob/main/CONTRIBUTING.md)
 - [x] I have considered the performance impact of these changes
 - [x] Suitable unit/system level tests have been added and they pass - automated test passed
 - [x] Documentation has been updated - Not required

### CODE REVIEW

**Review by ZJvandeWeg — APPROVED** [review]

**Comment by ZJvandeWeg** [comment]
Merging is pending the headshot update by @Yndira-FlowForge, otherwise it looks good to merge!

**Comment by Yndira-FlowForge** [comment]
I updated the image, but since the title is taking two lines, the cards next to this have a bigger overlay than they should
Screenshot 2023-06-13 at 12 43 20
Need to fix that before merging.

**Comment by Hisma** [comment]
The easy fix may be to just change my title to "developer advocate" and remove the rest of the detail. I do like having industrial automation in my title but it seems to break the flow.

File: src/_data/team/richard-meyer.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "order": 12,
+    "name": "Richard Meyer",
+    "title": "Developer Advocate - Manufacturing & Industrial Automation",
```
Comment (Yndira-FlowForge) [inline src/_data/team/richard-meyer.json:4]: "title": "Developer Advocate",

**Comment by Yndira-FlowForge** [comment]
> change my title to "developer advocate" and remove the rest of the detail

@Hisma just did that, we can work on making space for longer titles without breaking the flow later.

**Pull request merged**